setup.py

```diff
@@ -484,7 +505,7 @@ def mirror_files_into_torchgen():
 # all the work we need to do _before_ setup runs
 def build_deps():
     report("-- Building version " + version)
-
+    checkout_nccl()
```

Comment:
Could we make it conditional on `USE_CUDA`?

Reply:
Conditional USE_CUDA not always set. Maybe we should fix the USE_CUDA condition at the same time